support-frontend/assets/helpers/paymentIntegrations/readerRevenueApis.js

```diff
@@ -11,11 +11,11 @@ import { logPromise, pollUntilPromise } from 'helpers/promise';
 import { logException } from 'helpers/logger';
 import { fetchJson, getRequestOptions, requestOptions } from 'helpers/fetch';
 import trackConversion from 'helpers/tracking/conversions';
-import { type PaperFulfilmentOptions } from 'helpers/productPrice/fulfilmentOptions';
-import { type PaperProductOptions } from 'helpers/productPrice/productOptions';
+import type { FulfilmentOptions } from 'helpers/productPrice/fulfilmentOptions';
+import type { ProductOptions } from 'helpers/productPrice/productOptions';
```

Comment:
what's the difference? if any

Reply:
Outside the braces applies to everything within, inside only applies to the single type that follows the keyword